ui/src/app/tasks-jobs/tasks/task/task.component.html

```diff
@@ -16,40 +16,45 @@ <h1 class="page-title">
       [appRole]="['ROLE_SCHEDULE']"
       appFeature="schedules"
     >
-      Schedule
+      {{ 'tasks.task.schedule' | translate }}
     </button>
     <button type="button" class="btn btn-sm btn-secondary" grafanaDashboardTask [task]="task">Grafana Dashboard</button>
     <button type="button" class="btn btn-sm btn-outline-danger" (click)="destroy()" [appRole]="['ROLE_DESTROY']">
-      Destroy task
+      {{ 'tasks.task.destroy' | translate }}
     </button>
     <button type="button" class="btn btn-sm btn-outline-danger" (click)="cleanup()" [appRole]="['ROLE_DESTROY']">
-      Cleanup
+      {{ 'tasks.task.cleanup' | translate }}
     </button>
   </div>
 
   <div class="clr-row">
     <div class="clr-col-lg-6 clr-col-md-12 clr-col-12">
-      <app-view-card titleModal="Information" keyContext="task" name="{{ task.name }}" id="info">
+      <app-view-card
+        [titleModal]="'tasks.task.information' | translate"
+        keyContext="task"
+        name="{{ task.name }}"
+        id="info"
+      >
         <ng-template>
           <div class="block card-block-keyvalue">
             <div *ngIf="task.description" class="row">
-              <div class="key">Description</div>
+              <div class="key">{{ 'tasks.main.description' | translate }}</div>
               <div class="value">{{ task.description }}</div>
             </div>
             <div class="row">
-              <div class="key">Definition</div>
+              <div class="key">{{ 'tasks.main.definition' | translate }}</div>
               <div class="value">
                 <span class="dsl-text">{{ task.dslText }}</span>
               </div>
             </div>
             <div class="row">
-              <div class="key">Status</div>
+              <div class="key">{{ 'tasks.main.status' | translate }}</div>
               <div class="value">
                 <span class="{{ task.labelStatusClass() }}">{{ task.status }}</span>
               </div>
             </div>
             <div class="row">
-              <div class="key">Applications</div>
+              <div class="key">{{ 'tasks.task.applications' | translate }}</div>
```

Comment:
Texts like `Applications`, `Description`, `Status` and `Definition` are visible on other pages.
Can we move these to `commons` translations ?